packages/cms-data-sync/src/pages/pages.data-migration.ts

```diff
@@ -13,13 +14,16 @@ import { pagesQuery } from './pages.queries';
 
 type PageItem = NonNullable<FetchPagesQuery['queryPagesContents']>[number];
 
+const limiter = new RateLimiter({ tokensPerInterval: 5, interval: 'second' });
```

Comment:
i don't think you should be instantiating it every time - instead just export an instance with these settings and make sure to import that instance everywhere you need to make a request. Otherwise each of these requests work independently. Alternatively make sure to create the instance right at the beginning of the run-migrations file and then pass that along